TipCatDotNet.Api/Services/Payments/StripeAccountService.cs

```diff
@@ -87,11 +88,126 @@ async Task<Result> CreateRelatedAccount(string accountId)
             await _context.SaveChangesAsync(cancellationToken);
             _context.DetachEntities();
 
+            await SetStripeAccountActive(accountId, request.Id.Value, cancellationToken);
+
             return Result.Success();
         }
     }
 
 
+    public Task<Result> AddForAccount(TipCatData.HospitalityFacility.Account account, CancellationToken cancellationToken)
+    {
+        return Result.Success()
+            .Bind(CreateStripeAccount)
+            .Bind(SetStripeAccount);
+
+
+        async Task<Result<string>> CreateStripeAccount()
```

Comment:
Let's discuss that part on Monday. Look's like I don't understand the concept behind this method.